src/burn/engine/plan.cpp

```diff
@@ -1033,6 +1037,9 @@ extern "C" HRESULT PlanCachePackage(
 
     if (fBARequestedCache || NeedsCache(pPlan, pPackage))
     {
+        // The behavior for cache only mode is to do nothing on rollback, e.g. for subsequent install on demand scenarios.
```

Comment:
Doesn't this make it possible for packages to be in the cache without the bundle available to clean it out?